src/client.h

```diff
@@ -50,7 +53,9 @@ extern bool        g_bTSOverride;
 extern bool        g_bCacheM3U;
 extern bool        g_bCacheEPG;
 extern int         g_iEPGLogos;
+extern int         g_logoPathType;
 
 extern std::string PathCombine(const std::string &strPath, const std::string &strFileName);
 extern std::string GetClientFilePath(const std::string &strFileName);
 extern std::string GetUserFilePath(const std::string &strFileName);
+extern const std::string UrlEncode(const std::string &value);
```

Comment:
you can declare `UrlEncode` method where it's used i.e. in `PVRIptvData.cpp` file. no need to export it